### PR TITLE
[Keyboard] New Keyboard - Murphpad

### PR DESCRIPTION
**Pull request by kylemccreery**
## Description

Initial PR for adding the MurphPad numpad by MechWild. This is the default and VIA keymaps. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

*  None

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/mechwild/murphpad/keymaps/via/rules.mk [hunk 1]
```diff
@@ -0,0 +1,3 @@
+VIA_ENABLE = yes
+
+EXTRAFLAGS += -flto
```
Comment (fauxpark) [inline keyboards/mechwild/murphpad/keymaps/via/rules.mk:3]:
```suggestion
LTO_ENABLE = yes
```

File: keyboards/mechwild/murphpad/readme.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# MurphPad
+
+![murphpad](https://i.imgur.com/UmlpQ9p.jpg)
```
Comment (fauxpark) [inline keyboards/mechwild/murphpad/readme.md:3]:
```suggestion

![murphpad](https://i.imgur.com/UmlpQ9ph.jpg)
```

File: keyboards/mechwild/murphpad/rules.mk [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+ENCODER_ENABLE = yes		# Enable encoder
+OLED_DRIVER_ENABLE = yes	# Enable OLED Screen
```
Comment (fauxpark) [inline keyboards/mechwild/murphpad/rules.mk:24]:
```suggestion
ENCODER_ENABLE = yes        # Enable encoder
OLED_DRIVER_ENABLE = yes    # Enable OLED Screen
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]
Submodule changes need to be reverted before this can be merged.

**Comment by kylemccreery** [comment]
> Submodule changes need to be reverted before this can be merged.

Yep. This wasn't supposed to push. Not sure why it did.

**Comment by kylemccreery** [comment]
@fauxpark Sorry about that. Git still confuses me sometimes. I believe it is fixed now.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/mechwild/murphpad/rules.mk [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+ENCODER_ENABLE = yes        # Enable encoder
+OLED_DRIVER_ENABLE = yes	# Enable OLED Screen
```
Comment (fauxpark) [inline keyboards/mechwild/murphpad/rules.mk:24]:
```suggestion
OLED_DRIVER_ENABLE = yes    # Enable OLED Screen
```

**Review by fauxpark — CHANGES_REQUESTED** [review]
And some other stuff above.

**Comment by kylemccreery** [comment]
I think those changes were undone when I reverted to try and fix the submodule. My bad. Thanks for catching it @fauxpark

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**